src/modules/ROOT/pages/changelog/202303/00.json

```diff
@@ -37,18 +44,19 @@
         "That icon is clickable even though refresh will happen automatically based on the duration you have set."
       ],
       "why": [
-        "because users do not have to refresh the entire page to see new content on blocks."
+        "because users do not have to refresh the entire page to see new content on blocks and to give users more control over refreshing when auto-refreshing is not needed."
       ],
       "refs": [
-        "https://github.com/cortezaproject/corteza-js/pull/73/commits/1a77d99e54697f1bbe29622688996dc05a8c06c4"
+        "https://github.com/cortezaproject/corteza-js/pull/73/commits/1a77d99e54697f1bbe29622688996dc05a8c06c4",
+        "https://github.com/cortezaproject/corteza/pull/760/commits/954419bac6c3d19e3c5c46adf93d908528ad18df"
       ]
     },
     {
       "what": [
-        "auth Background Image card with a reactive image uploader form and a multiline text editor with CSS syntax highlighting on the System settings page."
+        "support for personalized auth screen."
```

Comment:
I think this is too vague.
Here, I'd point out we've simplified smaller (styling and background) changes with the new admin webapp additions.